.github/workflows/release.yml

```diff
@@ -22,6 +22,13 @@ jobs:
     runs-on: ubuntu-latest
     name: "Release a new version"
     steps:
+        - id: killswitch
+          if: ${{ !github.event.inputs.force }}
+          run: |
+            if curl -s "https://api.github.com/repos/$GITHUB_REPOSITORY/issues?state=open&labels=release-blocker" | grep -Pzvo '\[[\s\n\r]*\]'; then
```

Comment:
Perhaps it makes sense to add the GitHub token to this request too?

Reply:
Why do you think so? The token would only help with 2 issues (as far as I can tell):

1. If this repo was private
2. If there was any rate limiting concerns

I see none apply here. Do you have something else in mind that I am missing?

Reply:
Rate limiting, mostly. I remember Travis had a rate-limiting issue at some point when talking to GitHub, and since we already have the token in the environment, we could just use it to be on the safe side.
I know it's **GitHub** Actions, but they still must have a limited number of IPs in their disposal.

Reply:
from experience, in travis you pretty much hit the unauthenticated rate limits instantly when doing api requests on every master push, but this only runs per release. it *should* be fine but adding the token seems like a low-hanging fruit still:

```
        --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' \
```

https://docs.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token

Reply:
I'll pass for now until this becomes a real issue to keep things simpler. Thanks a lot for pointing this out though.